.chglog/config.yml

```diff
@@ -0,0 +1,20 @@
+style: github
+template: CHANGELOG.tpl.md
+info:
+  title: CHANGELOG
+  repository_url: https://github.com/pendulum-chain/spacewalk
+options:
+  commits:
+  commit_groups:
+  header:
+    pattern: "^(\\w*)[!]?\\:\\s(.*)$"
```

Comment:
I don't understand why we use this commit header pattern here. (what even is the commit header?)

Reply:
AFAIK the commit header is like the first line of your commit which can be 50 characters and the body would be the content you add in new lines.
But we don't need that anyways, so I removed it. Since we squash and merge every feature, it works fine to use the remaining config for merge commits.